packages/cli/api-importers/openapi/openapi-ir-to-fern-tests/src/__test__/__snapshots__/openapi/deepgram.json

```diff
@@ -10519,103 +10579,139 @@ channel:
-              },
-              {
-                "name": "Nova2Video",
-                "value": "nova-2-video",
-              },
-              {
-                "name": "Nova2Medical",
-                "value": "nova-2-medical",
-              },
-              {
-                "name": "Nova2Drivethru",
-                "value": "nova-2-drivethru",
-              },
-              {
-                "name": "Nova2Automotive",
-                "value": "nova-2-automotive",
-              },
-              "nova",
-              {
-                "name": "NovaGeneral",
-                "value": "nova-general",
-              },
-              {
-                "name": "NovaPhonecall",
-                "value": "nova-phonecall",
-              },
-              {
-                "name": "NovaMedical",
-                "value": "nova-medical",
-              },
-              "enhanced",
-              {
-                "name": "EnhancedGeneral",
-                "value": "enhanced-general",
-              },
-              {
-                "name": "EnhancedMeeting",
-                "value": "enhanced-meeting",
-              },
-              {
-                "name": "EnhancedPhonecall",
-                "value": "enhanced-phonecall",
-              },
-              {
-                "name": "EnhancedFinance",
-                "value": "enhanced-finance",
+            "discriminated": false,
+            "docs": undefined,
+            "encoding": undefined,
+            "inline": undefined,
+            "source": {
+              "openapi": "../openapi.yml",
+            },
+            "union": [
+              "literal<"nova-3">",
+              "literal<"nova-3-general">",
+              "literal<"nova-2">",
+              "literal<"nova-2-general">",
+              "literal<"nova-2-meeting">",
+              "literal<"nova-2-finance">",
+              "literal<"nova-2-conversationalai">",
+              "literal<"nova-2-voicemail">",
+              "literal<"nova-2-video">",
+              "literal<"nova-2-medical">",
+              "literal<"nova-2-drivethru">",
+              "literal<"nova-2-automotive">",
+              "literal<"nova">",
+              "literal<"nova-general">",
+              "literal<"nova-phonecall">",
+              "literal<"nova-medical">",
+              "literal<"enhanced">",
+              "literal<"enhanced-general">",
+              "literal<"enhanced-meeting">",
+              "literal<"enhanced-phonecall">",
+              "literal<"enhanced-finance">",
+              "literal<"base">",
+              "literal<"meeting">",
+              "literal<"phonecall">",
+              "literal<"finance">",
+              "literal<"conversationalai">",
+              "literal<"voicemail">",
+              "literal<"video">",
+              {
+                "docs": "Custom string if you've had a model trained by Deepgram",
+                "type": "string",
               },
```

Comment:
Discussed offline - this solution works really well for TypeScript, but it's terrible for other languages (e.g. Go, Java, etc). We should instead generate this type as an `enum` and a `string`. This will end up with a far better experience across the board.